packages/app-desktop/gui/Sidebar/hooks/useOnSidebarKeyDownHandler.ts

```diff
@@ -54,14 +54,13 @@ const useOnSidebarKeyDownHandler = (props: Props) => {
 			indexChange = -1;
 		} else if (event.code === 'ArrowDown') {
 			indexChange = 1;
-		} else if (event.code === 'Tab') {
+		} else if (event.code === 'Tab' && event.shiftKey) {
 			event.preventDefault();
 
-			if (event.shiftKey) {
-				void CommandService.instance().execute('focusElement', 'noteBody');
-			} else {
-				void CommandService.instance().execute('focusElement', 'noteList');
-			}
+			void CommandService.instance().execute('focusElement', 'noteBody');
```

Comment:
This focus override should also be removed, though perhaps in a follow-up pull request.